src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -47,6 +54,43 @@ class PythonToolRequirementsBase(Subsystem):
     default_lockfile_resource: ClassVar[tuple[str, str] | None] = None
     default_lockfile_url: ClassVar[str | None] = None
 
+    install_from_resolve = StrOption(
```

Comment:
It will still work as before if `pylint` is still a "tool lockfile". If you switch to using this new feature then presumably `pylint` is now the name of a "user resolve", in which case it should, again, work as before.